src/main/java/mate/academy/spring/config/WebAppInitializer.java

```diff
@@ -5,7 +5,7 @@
 public class WebAppInitializer extends AbstractAnnotationConfigDispatcherServletInitializer {
     @Override
     protected Class<?>[] getRootConfigClasses() {
-        return new Class[]{AppConfig.class, SecurityConfig.class};
+        return new Class[]{AppConfig.class};
```

Comment:
Why do you delete SecurityConfig.class from here??

Reply:
amended